db/dcrsqlite/sqlite.go

```diff
@@ -386,7 +386,11 @@ func (db *DB) RetrievePoolInfo(ind int64) (*apitypes.TicketPoolInfo, error) {
 	var hash, winners string
 	err := db.QueryRow(db.getPoolSQL, ind).Scan(&hash, &tpi.Size,
 		&tpi.Value, &tpi.ValAvg, &winners)
-	tpi.Winners = strings.Split(winners, ";")
+	if winners == "" {
+		tpi.Winners = make([]string, 0)
+	} else {
```

Comment:
The first part is already happening when `tpi` is initialized.  You just need `if winners != "" { tpi.Winners = ... }

Reply:
oh, nvm, I forgot what we talked about earlier.  nil slice is no good, we want zero length non-nil like you did.